website/docs/reference/warehouse-profiles/tidb-profile.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "TiDB Profile"
+---
+
+:::info Vendor-supported plugin
+
+Some core functionality may be limited. If you're interested in contributing, check out the source code for each repository listed below.
```

Comment:
perhaps it's worth linking back to your readme about what is and isn't supported?

Reply:
Thanks to @dataders for review. I have linked the relevant documentation.